docs/feature-store/feature-sets.md

```diff
@@ -43,13 +43,33 @@ Example:
 stocks_set = FeatureSet("stocks", entities=[Entity("ticker")])
 ```
 
-To learn more about Feature Sets go to {py:class}`~mlrun.feature_store.FeatureSet`.
 
-```{admonition} Note 
-Feature sets can also be created in the UI. To create a feature set:
+### Create a feature set without ingesting its data
+
+You can define and register a feature set, and use it in a feature vector, without ingesting its data. This method is 
```

Comment:
You can define and register a feature set (and use it in a feature vector) without ingesting its data into mlrun offline targets.
The use-case for this is when you have a large amount of data at some remote storage that is ready to be consumed by a model-training pipeline.
When this feature is enabled on a feature-set, data is not saved to offline target during ingestion. Instead, when a get_offline_features is called on a vector containing that feature-set, that data will be read directly from the source.
Note that online targets are still ingested, and their value represents a time-slice of the offline source.
Note also that transformations are not allowed when this feature is enabled: No computation graph, no aggregations etc.
Enable this feature by including `passthrough=True` in the feature set definition. All three ingestion engines (Storey, Spark, Pandas) are supported, as well as retrieval engines "local" and "spark".